App.UserSupport/Models/Message.cs

```diff
@@ -7,16 +7,19 @@ namespace App.UserSupport.Models
     public class Message
     {
         private Client clientHandling;
-
+        public int Id { get; set; }
         public string mess { get; set; }
-        public Message(Client client,string mess)
+        public Message(Client client,string mess,int id)
```

Comment:
You always know Id of the message when calling constructor ?

Reply:
It's just ovveride constructor for realization add message in db in class Handling

Reply:
You don`t need to specify Ids by yourself, Ids are setted by EF automatically, if you configure Id properties in a right way